### PR TITLE
CMS-1394 Fix meta languages bug

### PR DESCRIPTION
**Pull request by sdonk**

### CODE REVIEW

File: conf/settings.py [hunk 1]
```diff
@@ -180,12 +180,10 @@
     ('en-gb', 'English', 'English'),
     ('de', 'German', 'Deutsch'),
     ('ja', 'Japanese', '日本語'),
-    ('ru', 'Russian', 'Russian'),
     ('zh-hans', 'Simplified Chinese', '简体中文'),
     ('fr', 'French', 'Français'),
     ('es', 'Spanish', 'español'),
     ('pt', 'Portuguese', 'Português'),
-    ('pt-br', 'Portuguese (Brazilian)', 'Português Brasileiro'),
```
Comment (sdonk) [inline conf/settings.py]: The above languages are unused

File: core/models.py [hunk 2]
```diff
@@ -271,12 +271,26 @@ def translated_languages(self):
...
+        # published in CMS. I'm forcing en-gb in the list to avoid this issue.
+        # This is diffucult to test both programmatically and manually and it's
+        # a corner case so I'm leaving the next line effectively untested.
+        # A manual test would have the following steps:
+        # 1) Add a page model
+        # 2) Create an instance of the above page model in wagtail
+        # 3) Add at least one new mandatory field to the model
+        # 4) Migrate CMS
+        # 5) Open the page on the UI client without adding the new content
+        translated_languages = ['en-gb']
```
Comment (sdonk) [inline core/models.py]: I line fix and 14 lines comment!

File: requirements_test.in [hunk 3]
```diff
@@ -5,7 +5,6 @@ pytest-cov
 pytest-django
 pytest-sugar
 flake8
-pytest-catchlog
```
Comment (sdonk) [inline requirements_test.in]: Catchlog is now part of pytest core, removed
